index.js

```diff
@@ -18,4 +18,242 @@ window.onclick = function exit(event) {
+    url: './images/project2.png',
+    title: 'Keeping track of hundreds of components',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
+    tecnologies: {
+      tech1: 'Ruby on rails',
+      tech2: 'css',
+      tech3: 'JavaScript',
+      tech4: 'html',
+    },
+    linklive: 'https://brhanuhailu.github.io/',
+    linksource: 'https://github.com/brhanuhailu/portifolio',
+  },
+  {
+    id: 3,
+    url: './images/project4.png',
+    title: 'Build a CryptoCurrency Tracker App',
+    description: 'The idea is great and you can actually put this app into App Store or Google PlayStore as React Native allows you to create cross-platform apps. You can use React.js, React Native, and Firebase to build the course, where Firebase can be used ',
+    tecnologies: {
+      tech1: 'Ruby on rails',
+      tech2: 'css',
+      tech3: 'JavaScript',
+      tech4: 'html',
+    },
+    linklive: 'https://brhanuhailu.github.io/',
+    linksource: 'https://github.com/brhanuhailu/portifolio',
+  },
+  {
+    id: 4,
+    url: './images/project1.png',
+    title: 'Keeping track of hundreds of components',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.Lorem Ipsum is simply dummy text of the printing and typesetting industry',
+    tecnologies: {
+      tech1: 'Ruby on rails',
+      tech2: 'css',
+      tech3: 'JavaScript',
+      tech4: 'html',
+    },
+    linklive: 'https://brhanuhailu.github.io/',
+    linksource: 'https://github.com/brhanuhailu/portifolio',
+  },
+  {
+    id: 5,
+    url: './images/project2.png',
+    title: 'Keeping track of hundreds of components',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. ths helps you to avoid plagiarisum and support to respect the intellectual property right',
+    tecnologies: {
+      tech1: 'Ruby on rails',
+      tech2: 'css',
+      tech3: 'JavaScript',
+      tech4: 'html',
+    },
+    linklive: 'https://brhanuhailu.github.io/',
+    linksource: 'https://github.com/brhanuhailu/portifolio',
+  },
+];
+const workdetailsSection = (content) => `
+    <span class="close-details">&times;</span>
+    <img class="imag-details" src=${WorkDatas[content].url} alt="Image details">
+    <h2 class="project-name">${WorkDatas[content].title}</h2>
+    <ul class="detail-tags">
+        <li><a href="">${WorkDatas[content].tecnologies.tech1}</a></li>
+        <li><a href="">${WorkDatas[content].tecnologies.tech2}</a></li>
+        <li><a href="">${WorkDatas[content].tecnologies.tech3}</a></li>
+        <li><a href="">${WorkDatas[content].tecnologies.tech4}</a></li>
+    </ul>
+    <p class="project-desc">${WorkDatas[content].description}</p>
+    <div class="btn-detail">
+        <a href="${WorkDatas[content].linklive}">See Live <i class="fa fa-external-link"></i></a>
+        <a href="${WorkDatas[content].linksource}"> See Source <i class="fa fa-github"></i></a>
+    </div>
+ `;
```

Comment:
- [ ] You have managed to implement the dynamic popup. Though, there are a few designs to make to stick to the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A2702&t=x6C9OeUy4GbFv0mj-0). 
   - Kindly verify the image path so that the image of the project shows. This will make your project more professional. 
   - The popup card should be centered with the `blur` background. I suggest that you make sure to implement the blurred background and that the card be centered in the window.
   - In addition, I suggest that you change the color and background of `.close-details` to match the one if the Figma template. This is because the close button is not visible and I have realized that you gave it a white color.
   - Please ensure that the same above is applied on desktop version. In addition, ensure that the padding of the project name is reduced to match the Figma design ☺️ .
   - Lastly ensure that the the height is greater that the width of the modal popup on desktop.

Your mobile design  😎 
![popup](https://user-images.githubusercontent.com/99394374/216000888-77c46102-4bcc-49d6-859b-62dc04f0b6cb.PNG)

Figma design 😎 
![popup fig](https://user-images.githubusercontent.com/99394374/216000958-6b7e277d-6310-4187-8aa8-66465b86dbad.PNG)

Your desktop design 😎 
![desk](https://user-images.githubusercontent.com/99394374/216002056-a77a7a06-9a2a-42d9-b886-e5a37a8ae63f.PNG)

Reply:
Fixed